src/dapla_pseudo/v1/ops.py

```diff
@@ -106,6 +107,10 @@ def pseudonymize(
             content_type = Mimetypes(magic.from_buffer(dataset.read(2048), mime=True))
             dataset.seek(0)
             file_handle = dataset
+        case AbstractFileSystem():
```

Comment:
Would be good to write a little test to cover this case.

Reply:
Yep, I'll add a test for it